usr/share/rear/prep/DP/default/450_check_dp_client_configured.sh

```diff
@@ -10,9 +12,14 @@ Log "Backup method is DP: check Data Protector requirements"
 test -x $VBDA || Error "Cannot execute $VBDA
 Install Data Protector Disk Agent (DA component) on the client."
 
-test -x $OMNIR || Error "Cannot execute $OMNIR
-Install Data Protector User Interface (CC component) on the client."
+if [ $ARCH == "Linux-i386" ] || [ $ARCH == "Linux-x86_64" ] || [ $ARCH == "Linux-ia64" ]; then
```

Comment:
We have the arch-dependent sub directories to avoid these sort of checks, maybe they make the coder simpler?